.github/workflows/builds.yml

```diff
@@ -139,8 +125,18 @@ jobs:
         run: |
           Remove-Item bin/* -Include *.exp,*.lib,*.pdb -Force
       - name: Upload Artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: godot-cpp-template
+          name: godot-cpp-template-${{ matrix.platform }}-${{ matrix.arch }}-${{ matrix.float-precision }}-${{ matrix.target-type }}
           path: |
             ${{ github.workspace }}/bin/**
+  merge:
+    runs-on: ubuntu-20.04
+    needs: build
+    steps:
+      - name: Merge Artifacts
+        uses: actions/upload-artifact/merge@v4
+        with:
+          name: godot-cpp-template
+          pattern: godot-cpp-template-*
+          delete-merged: true
```

Comment:
Does this merge command mean that all artifacts will be in one folder zipped? Because this may not be always desired and could be added in comments with an explanation to leave open to the end user of the template to use or not to use

Reply:
I believe it has the same functionality as the old upload artifacts uploading files into one artifact name.

Reply:
Yeah in that case they will be merged into one folder then 🤔 Hmm I suppose that can stay then to have them all in one download. Could you make a comment above to document that it combines all artifacts into one available download then and if this command is commented out, the artifacts will be downloadable separately?